modules/cnf-scheduling-workload-onto-worker-with-real-time-capabilities.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+[id="cnf-scheduling-workload-onto-worker-with-real-time-capabilities_{context}"]
+= Scheduling a workload onto a worker with real-time capabilities
```

Comment:
Might be better as:
Scheduling a low latency workload onto a worker with real-time capabilities